internal/server/v1/observations/golden/bulk_point/preferred_latest.json

```diff
@@ -210,9 +210,9 @@
           "entity": "geoId/0649670",
           "points_by_facet": [
             {
-              "date": "2022-11",
-              "value": 940.733829,
-              "facet": "4026503442"
+              "date": "2021",
```

Comment:
@shifucun I don't fully understand this. Is the preference always supposed to be for P1Y data? There is P1M as well: https://autopush.datacommons.org/tools/timeline#place=geoId%2F0649670&statsVar=Count_Person_Unemployed&mh=%7B%22Count_Person_Unemployed%22%3A%22324358135%22%7D Is this expected?

Reply:
There is ranking for some of the facets: https://github.com/datacommonsorg/mixer/blob/master/internal/server/ranking/ranking.go#L44. So preferred is from this ranking if present, otherwise, we look at the facets with most/more recent data.